config/application.rb

```diff
@@ -11,8 +11,5 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
-
-    config.eager_load_paths << Rails.root.join('app', 'graph', 'types')
-    config.eager_load_paths << Rails.root.join('app', 'graph', 'fields')
```

Comment:
I saw a lot of questions about Rails not loading types and fields automatically so I've removed these eager loads and added modules so Rails autoloader can do it's job